<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 40, 49);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Signin_IDEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 40, 49);
font: 87 12pt &quot;Arial Black&quot;;
color: rgb(238, 238, 238);</string>
   </property>
   <property name="text">
    <string>UserName</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Signin_PWEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(34, 40, 49);
font: 87 12pt &quot;Arial Black&quot;;
color: rgb(238, 238, 238);</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>64</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 238);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>64</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 211, 105);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 24pt &quot;Arial Black&quot;;
color: rgb(255, 211, 105);</string>
   </property>
   <property name="text">
    <string>Sign In</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius : 10px;
	font: 87 11pt &quot;Arial Black&quot;;
	background-color: rgb(238, 238, 238);
}

QPushButton:pressed{
	border-radius : 10px;
	font: 87 11pt &quot;Arial Black&quot;;
	
	background-color: rgb(200, 200, 200);
}</string>
   </property>
   <property name="text">
    <string>CHECK ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 238);
font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>PLZ CHECK ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius : 10px;
	font: 87 11pt &quot;Arial Black&quot;;
	background-color: rgb(238, 238, 238);
}

QPushButton:pressed{
	border-radius : 10px;
	font: 87 11pt &quot;Arial Black&quot;;
	
	background-color: rgb(200, 200, 200);
}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius : 10px;
	font: 87 11pt &quot;Arial Black&quot;;
	background-color: rgb(238, 238, 238);
}

QPushButton:pressed{
	border-radius : 10px;
	font: 87 11pt &quot;Arial Black&quot;;
	
	background-color: rgb(200, 200, 200);
}</string>
   </property>
   <property name="text">
    <string>CANCLE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
